addons/api/addon/transforms/string-array.js

```diff
@@ -0,0 +1,13 @@
+import Transform from '@ember-data/serializer/transform';
+
+export default class StringArrayTransform extends Transform {
+  deserialize(serialized) {
+    const stringValues = serialized || [];
+    return stringValues.map((value) => ({ value }));
+  }
+
+  serialize(deserialized) {
```

Comment:
I don't see these being used outside of the tests include in the PR. Are these being added for future use?

Reply:
You're right.  This is here just for completeness.  Not used yet.

Reply:
But it might be used by other fragment string arrays.  Won't know until they are all refactored.